Makefile

```diff
@@ -53,9 +53,12 @@ install:
 
 	cp -Ra dbicdh "$(DESTDIR)"/usr/share/openqa/dbicdh
 
-test:
-	OPENQA_CONFIG= prove -r
-	./script/tidy --check
+
+.PHONY: check_style
+check_style:
 	PERL5LIB=lib/perlcritic:$$PERL5LIB perlcritic --gentle --include Perl::Critic::Policy::HashKeyQuote lib
 
+test: check_style
+	OPENQA_CONFIG= prove -r
+
 .PHONY: all install test
```

Comment:
Because it doesn't scale well and eventually people will forget to add something to .PHONY. What about (1) splitting it and moving all .PHONY definition to each target? Or (2) should I add it to L64?